src/avt/Queries/Misc/avtDirectDatabaseQOTFilter.h

```diff
@@ -35,7 +35,7 @@ class vtkUnstructuredGrid;
 //      but there are some significant drawbacks that must be noted:
 //
 //          1. This filter can only retrieve "original" data, meaning that
-//             the query will be performed before any other plots and filters
+//             the query will be performed before any other plots and filter
 //             are applied (excluding the expression filter).
```

Comment:
?

Reply:
Good question.  Probably an overzealous use of whitespace elimination. I thought I would erase all trailing 's' too.  LOL. Will fix that.